docs/spl-docs/bin-command.md

```diff
@@ -65,7 +65,7 @@ This value is disregarded if it is less than maximum value present in the data d
 This option applies only to numerical data and does not apply to time-based data.
 
 #### \<aligntime\>
-**Syntax:** `<absolute-time> | <relative-time> | <N>`\
+**Syntax:** `<absolute-time>` or `<relative-time>` or `<N>`\
```

Comment:
nit: the way you're giving options is inconsistent with what you did on line 27 (where you only used commas). The best is probably a combination (all commas except use `or` for the last):
```
**Syntax:** `<absolute-time>`, `<relative-time>` or `<N>`\
```

Some other places need to change similarly, but lines 49 and 94 are good how you have them

Reply:
I used commas there because you can multiple of them. For portions where or is present, you can only enter any one of them. What do you think should we have same kind of format whether they can use combinations or enter one of them ?

Reply:
From your overall syntax for `bin`:
```
**bin**\
[\<bin-options\>...]\
**\<field\>** [AS \<newfield\>]
```
I think `[\<bin-options\>...]\` makes it clear that you can have multiple options. So I think evern for line 27 where you're giving the valid options for a `bin-option` you can do commas and a final `or`